internal/cmd/kafka/command_cluster_describe.go

```diff
@@ -178,9 +177,9 @@ func (c *clusterCommand) getTopicCountForKafkaCluster(cluster *cmkv2.CmkV2Cluste
 		return 0, err
 	}
 
-	topics, httpResp, err := kafkaREST.CloudClient.ListKafkaTopics(cluster.GetId())
+	topics, _, err := kafkaREST.CloudClient.ListKafkaTopics(cluster.GetId())
 	if err != nil {
-		return 0, kafkarest.NewError(kafkaREST.CloudClient.GetUrl(), err, httpResp)
+		return 0, err
```

Comment:
Doesn't this change make the error less detailed?

Reply:
Doesn't that mean you're missing the http in the URL?

Reply:
Yeah, but it's a red herring. The url itself is apparently empty for these private link setups, so it's sending requests to an empty url with the topics endpoint tacked on at the end, which is why it reports an empty protocol scheme.

Reply:
huh, maybe we should add some logic where it doesn't send the request if the URL is `""`?

Reply:
Ok I think we should just proceed for now to get the functionality in, and if we can get confirmation that it's just a missing hostname we optimize later.